packages/browser-wallet/src/background/recovery.ts

```diff
@@ -165,12 +220,34 @@ async function performRecovery({ providers, ...recoveryInputs }: Payload) {
     }
 }
 
-export const recoveryHandler: ExtensionMessageHandler = (msg) => {
-    const respond = async (result: RecoveryBackgroundResponse) => {
-        await openWindow();
-        bgMessageHandler.sendInternalMessage(InternalMessageType.RecoveryFinished, result);
-    };
-    performRecovery(msg.payload)
-        .then((added) => respond({ status: BackgroundResponseStatus.Success, added }))
-        .catch((e) => respond({ status: BackgroundResponseStatus.Error, reason: e.toString() }));
-};
+export function startRecovery() {
+    chrome.alarms.create(RECOVERY_ALARM_NAME, { delayInMinutes: 0 });
```

Comment:
From the documentation it says:
```
That is, setting delayInMinutes or periodInMinutes to less than 1 will not be honored and will cause a warning. 
```
so perhaps this should not be set to `0`.

Reply:
Yeah, I guess it only works properly now because it is unpacked, rewriting this.